examples/cms-builder-io/postcss.config.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  plugins: [
+    'tailwindcss',
```

Comment:
Could we update all deps and remove this custom config to be aligned with the tailwind default recommendation?